src/app/tokens/api/apis.token.ts

```diff
@@ -0,0 +1,11 @@
+import { InjectionToken } from '@angular/core';
+import type { ApiConfiguration } from './api.configuration';
+
+export const APIS_TOKENS: InjectionToken<Record<AvailableApis, ApiConfiguration>> = new InjectionToken<
+  Record<AvailableApis, ApiConfiguration>
+>('apis.configuration');
+
+export enum AvailableApis {
```

Comment:
```suggestion
export enum Api {
```
Je trouve que c'est pas mal d'éviter les pluriels dans les noms des énumérations, car ce n'est pas une collection.
Et je trouve `Available` un peu lourd à l'usage dans le code des requêtes.